modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -49,4 +49,60 @@
       end
     end
   end
+
+  factory :auto_established_claim_without_flashes, class: 'ClaimsApi::AutoEstablishedClaim' do
+    id { SecureRandom.uuid }
+    status { 'pending' }
+    source { 'oddball' }
+    evss_id { nil }
+    auth_headers { { test: ('a'..'z').to_a.shuffle.join } }
+    form_data do
+      json = JSON.parse(File.read("#{::Rails.root}/modules/claims_api/spec/fixtures/form_526_no_flashes_json_api.json"))
+      json['data']['attributes']
+    end
+    flashes { form_data.dig('veteran', 'flashes') }
+    special_issues do
+      if form_data['disabilities'].present? && form_data['disabilities'].first['specialIssues'].present?
```

Comment:
I'm prob just missing something, but will this `if` ever be true?
Looks like the `disabilities.specialIssues`defined in `form_526_no_flashes_json_api.json` is an empty array.

Reply:
def not missing anything, just a copy/paste error. thanks for catching it!